CRM/Upgrade/Incremental/sql/4.6.9.mysql.tpl

```diff
@@ -13,3 +13,44 @@ ADD COLUMN `created_id` int(10) unsigned DEFAULT NULL COMMENT 'FK to contact tab
 ADD COLUMN `owner_id` int(10) unsigned DEFAULT NULL COMMENT 'FK to contact table.',
 ADD CONSTRAINT `FK_civicrm_report_instance_created_id` FOREIGN KEY (`created_id`) REFERENCES `civicrm_contact` (`id`) ON DELETE SET NULL,
 ADD CONSTRAINT `FK_civicrm_report_instance_owner_id` FOREIGN KEY (`owner_id`) REFERENCES `civicrm_contact` (`id`) ON DELETE SET NULL;
+
+-- CRM-16907
+SELECT @navMaxWeight := MAX(ROUND(weight)) from civicrm_navigation WHERE parent_id IS NULL;
+
+-- Add "support" menu if it's not there already
+INSERT INTO civicrm_navigation (domain_id, label, name, is_active, weight)
+SELECT * FROM (SELECT {$domainID} as domain_id, 'Support' as label, 'Help' as name, 1 as is_active, @navMaxWeight + 1 as weight) AS tmp
+WHERE NOT EXISTS (
+SELECT name FROM civicrm_navigation WHERE name = 'Help' AND domain_id = {$domainID}
+) LIMIT 1;
+
+SELECT @adminHelplastID := id FROM civicrm_navigation WHERE name = 'Help' AND domain_id = {$domainID};
+
+UPDATE civicrm_navigation
+SET name = 'Support', label = '{ts escape="sql" skip="true"}Support{/ts}'
+WHERE id = @adminHelplastID;
+
+DELETE FROM civicrm_navigation where parent_id = @adminHelplastID;
+
+INSERT INTO civicrm_navigation
+( domain_id, url, label, name, permission, permission_operator, parent_id, is_active, has_separator, weight )
+VALUES
+( {$domainID}, 'http://civicrm.org/get-started?src=iam',        '{ts escape="sql" skip="true"}Get started{/ts}',        'Get started',        NULL, 'AND', @adminHelplastID, '1', NULL, 1 ),
+( {$domainID}, 'http://civicrm.org/documentation?src=iam',      '{ts escape="sql" skip="true"}Documentation{/ts}',      'Documentation',      NULL, 'AND', @adminHelplastID, '1', NULL, 2 ),
+( {$domainID}, 'http://civicrm.org/ask-a-question?src=iam',     '{ts escape="sql" skip="true"}Ask a question{/ts}',     'Ask a question',     NULL, 'AND', @adminHelplastID, '1', NULL, 3 ),
+( {$domainID}, 'http://civicrm.org/experts?src=iam',            '{ts escape="sql" skip="true"}Get expert help{/ts}',    'Get expert help',    NULL, 'AND', @adminHelplastID, '1', NULL, 4 ),
+( {$domainID}, 'http://civicrm.org/about?src=iam',              '{ts escape="sql" skip="true"}About CiviCRM{/ts}',      'About CiviCRM',      NULL, 'AND', @adminHelplastID, '1', 1, 5 ),
+( {$domainID}, 'http://civicrm.org/register-your-site?src=iam&sid={ldelim}sid{rdelim}', '{ts escape="sql" skip="true"}Register your site{/ts}', 'Register your site', NULL, 'AND', @adminHelplastID, '1', NULL, 6 ),
+( {$domainID}, 'http://civicrm.org/become-a-member?src=iam&sid={ldelim}sid{rdelim}',      '{ts escape="sql" skip="true"}Join CiviCRM{/ts}',       'Join CiviCRM',       NULL, 'AND', @adminHelplastID, '1', NULL, 7 );
```

Comment:
Ah, good point.

It probably doesn't matter too much... but I think that DELETE might remove anything that sysadmin has put under "Help" manually. Maybe tighten it like this?

```sql
DELETE FROM civicrm_navigation WHERE parent_id = @adminHelplastID AND url LIKE "http://civicrm.org%"
```